tests/pages/firefox/os/tv.py

```diff
@@ -0,0 +1,65 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage, FirefoxBasePageRegion
+
+
+class TVPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/os/devices/tv'
+
+    _next_button_locator = (By.CSS_SELECTOR, '.pager-next')
+    _previous_button_locator = (By.CSS_SELECTOR, '.pager-prev')
+    _screens_locator = (By.CSS_SELECTOR, '.pager-page')
+    _thumbnails_locator = (By.CSS_SELECTOR, '.pager-tabs a')
+
+    @property
+    def screens(self):
+        return [Screens(self.base_url, self.selenium, root=el) for el in
+                self.find_elements(self._screens_locator)]
+
+    @property
+    def thumbnails(self):
+        return [Thumbnails(self.base_url, self.selenium, root=el) for el in
+                self.find_elements(self._thumbnails_locator)]
+
+    def click_next(self):
+        self.find_element(self._next_button_locator).click()
+
+    def click_previous(self):
+        self.find_element(self._previous_button_locator).click()
+
+    @property
+    def is_next_enabled(self):
+        return self.find_element(self._next_button_locator).is_enabled()
+
+    @property
+    def is_previous_enabled(self):
+        return self.find_element(self._previous_button_locator).is_enabled()
+
+
+class Screens(FirefoxBasePageRegion):
+
+    _image_locator = (By.TAG_NAME, 'img')
+    _next_button_locator = (By.CSS_SELECTOR, '.pager-next')
+    _previous_button_locator = (By.CSS_SELECTOR, '.pager-prev')
+
+    @property
+    def is_displayed(self):
+        self.wait.until(lambda s: self.is_element_displayed(self._image_locator))
+        return self.is_element_displayed(self._image_locator)
+
+
+class Thumbnails(FirefoxBasePageRegion):
+
+    _link_locator = (By.TAG_NAME, 'a')
+
+    @property
+    def is_selected(self):
+        return self._root.get_attribute('class') == 'selected'
+
+    def click(self):
+        self._root.click()
```

Comment:
I think we should try to avoid calling `_root`, although I do think we may need to have a way to interact with the root element. In most cases the root element would be a container and not something we'd directly interact with. In this case, looking at the page source, I think the `Thumbnails` root locator would be `.pager-tabs li` and then the `_link_locator` above would be used in the `click` and `is_selected` methods.